documentation/src/docs/security/pravega-security-encryption.md

```diff
@@ -0,0 +1,35 @@
+<!--
+Copyright (c) 2017 Dell Inc., or its subsidiaries. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+-->
+# Pravega Encryption
+## Encryption of data in flight 
+Pravega ensures that all the data in flight can be passed by applying encrytion.
+The different [channels](https://github.com/pravega/pravega/wiki/PDP-23:-Pravega-security----encryption-and-Role-Based-Access-Control#b-encryption-of-data-in-flight-over-network-and-in-tier-1)
+can be configured with TLS and encryption can be enabled for them.
+### Certificate Management
+Pravega expects administrators and users to create and manage certificate creation, deployment and management. 
+Pravega provides various configuration parameters using which certificates for different communication channels can be specified.
+### Encrypted data flow between Pravega client and Pravega controller and segmentstore
+Pravega uses same certificate to interact with the controller and segmentstore. The certificates needs to be mentioned specifically on the client and the server machine.
+
+**Note:** These certificates are not loaded from the truststore.
+
+### Encrypted data flow between Pravega and Tier 1 (Apache Bookkeeper)
+Pravega segmentstore uses Apache Bookkeeper as Tier 1 storage. Apache Bookkeeper supports JKS based truststore. Segmentstore uses JKS based truststore to interact with it. 
+The configurations can be found [here](pravega-security-configurations.md#pravega-segmentstore).
+
+### Encrypted access to Apache Zookeeper
+This implementation is still in [progress](https://github.com/pravega/pravega/issues/2034). 
+
+## Encryption of data at REST
```

Comment:
I don't understand `data at REST`. The capitalized `REST` seems to imply the controller REST API, but I don't know whether this is referring to something else.